src/runtime/tests/schema-test.ts

```diff
@@ -444,6 +444,21 @@ describe('schema', () => {
     assert.strictEqual(schema.fields.custom.type, 'Bytes');
   });
 
+  it('handles multi named aliased schemas with extensions', async () => {
+    const manifest = await Manifest.parse(`
+      alias schema Event Occurrence as EventAlias
+        Text name
+        
+      schema Accident extends EventAlias
+        Number financialCost
+      
+      schema Crisis extends Accident`);
+    
+    assert.deepEqual(manifest.findSchemaByName('EventAlias').names, ['Event', 'Occurrence']);
+    assert.deepEqual(manifest.findSchemaByName('Accident').names, ['Accident', 'Event', 'Occurrence']);
+    assert.deepEqual(manifest.findSchemaByName('Crisis').names, ['Crisis', 'Accident', 'Event', 'Occurrence']);
```

Comment:
can you, please, also add an assert to test the fields? thanks.

Reply:
Done :)